src/fromager/commands/bootstrap.py

```diff
@@ -24,8 +24,8 @@ def _get_requirements_from_args(toplevel, requirements_file):
 
 
 @click.command()
-@click.option('--variant', default='cpu',
-              help='the build variant name')
+@click.option('--variant', default='cpu', help='the build variant name',
+              type=click.Choice(['cpu', 'cuda-ubi9', 'gaudi-ubi9', 'rocm-ubi9']))
```

Comment:
These are the variants for what we're building for RHEL AI, but other people may want to build other variants.

I had considered looking for variants defined in the settings file, since that would be under the control of the user running fromager. What do you think about adding a section to the settings file or trying to derive it from that file in some other way?

For example, right now the file has something like 

```
pre_built:
  variant:
    - package1
    - package2
```

It might be better to flip the nesting levels of `pre_built` and `variant`, I don't know?

Reply:
Fantastic idea! I didn't consider open source concerns so the code should derive from the settings file and this will also allow us to still set this. Ill proceed with this but raise this again if I come across a better implementation. In the mean time I will convert this to a draft until I can re-implement this, starting with sequence download.